src/apollo/cache/colony/tokens.ts

```diff
@@ -19,7 +19,7 @@ const tokensFieldCache = {
           token: {
             __typename: 'Token',
             ...DEFAULT_NETWORK_TOKEN,
-            id: ADDRESS_ZERO,
+            tokenAddress: ADDRESS_ZERO,
```

Comment:
I'm not too sure what having `id` instead of `tokenAddress` broke, but it does look intentional especially if you consider the comment next to the `UnaliasedColonyTokensItem` type:

<img width="699" alt="image" src="https://github.com/JoinColony/colonyCDapp/assets/112586815/e67d276c-c391-46b3-9715-7cdbc781292e">

Could you address that as a whole, and either remove the type and the comment or update it?

Reply:
I will revert this for now so I don't inflict a lot of potential breaking changes :D
The cache is also already hardwired to work with the `id` field.